enginetest/queries/column_default_queries.go

```diff
@@ -775,4 +775,70 @@ var ColumnDefaultTests = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "column default normalization: int column rounds",
```

Comment:
I think we should add additional tests for:
- An unquoted default value becomes single-quoted in the `show create` statement.
- A double quoted default value becomes single-quoted in the `show create` statement.
- An expression that wraps a literal (`i int default (1)`) is preserved and not normalized.